rate_control.md

```diff
@@ -1,5 +1,31 @@
 ---
-title: Rate Control
+title: Rate Control (WIP)
 ---
 
-# Rate Control
+# Rate Control (WIP)
+
+## Overview
+
+The rate control protocol is the key part of the ultimate goal of UCSMA, which attempts to maintain fairness among different nodes in an ad-hoc wireless network. There are two parts of the rate control protocol:
+
+1. buffer enqueue delay: a packet will need to wait a certain time interval (positively proportional to the current buffer size) before being enqueued into the transmission buffer; 
+2. contention window adjustment: we set lower CW for nodes with fuller buffer to allow their packets to be send our quicker.
```

Comment:
"we set lower CW for nodes with fuller buffer to allow their packets to be send our quicker."
"we set a lower CW for nodes with fuller buffers to allow their packets to be sent out quicker."